lib/kitchen/directions/bake_chapter_introductions/v1.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeChapterIntroductions
+  class V1
+    def bake(book:, bake_chapter_objectives:, bake_chapter_outline:)
+      book.chapters.each do |chapter|
+        introduction_page = chapter.introduction_page
+
+        introduction_page.search("div[data-type='description']").trash
+        introduction_page.search("div[data-type='abstract']").trash
+
+        title = introduction_page.title.cut
+        title.name = 'h2'
+        Kitchen::Directions::MoveTitleTextIntoSpan.v1(title: title)
+
+        intro_content = introduction_page.search(
+          "> :not([data-type='metadata']):not(.splash):not(.has-splash)"
+        ).cut
+
+        chapter_objectives_html =
+          if bake_chapter_objectives
+            Kitchen::Directions::BakeChapterIntroductions.bake_chapter_objectives(
+              chapter: chapter
+            )
+          else
+            ''
+          end
+
+        final_section =
+          if bake_chapter_outline && bake_chapter_objectives
+            Kitchen::Directions::BakeChapterIntroductions.bake_chapter_outline(
+              chapter_objectives_html: chapter_objectives_html
+            )
+          else
+            chapter_objectives_html
+          end
```

Comment:
I'm not sure if I understand this part correctly :thinking: Why do we need `bake_chapter_objectives:` parameter? First version in only for Chapter Outline, that's right? It exists or not. And if it exists it always contain default Chapter Objectives (defined in `BakeChapterObjectives`). 

I'm wondering if something like that could work:

```
if bake_chapter_outline
  chapter_objectives_html = Kitchen::Directions::BakeChapterIntroductions.bake_chapter_objectives(
              chapter: chapter
            )
  chapter_outline = Kitchen::Directions::BakeChapterIntroductions.bake_chapter_outline(
              chapter_objectives_html: chapter_objectives_html
            )
end
```
and in the end:

```
introduction_page.append(child:
          <<~HTML
            <div class="intro-body">
              #{chapter_outline if bake_chapter_outline}
              <div class="intro-text">
                #{title.paste}
                #{intro_content.paste}
              </div>
            </div>
          HTML
        )
```

Did I miss any case?

Reply:
I talked to Alina and she says that there are book that do have one or the other, so we might run into books needing chapter objectives but not chapter outline

Reply:
Yes, but in situation when there is Chapter Objectives but there is no Chapter Outline we are using V2. Do we need this in V1? I guess all previous (released) books have Chapter Outline. First book that has only Chapter Objectives was one that Mary Jo was working on and now you have another book, that's right? If yes, then maybe we can remove this case from V1 :thinking: